src/modules/checklists/v1/controllers/ChecklistController.ts

```diff
@@ -144,15 +144,15 @@ export class ChecklistController extends BaseController {
 
             await new ChecklistRepository().update(checklist);
 
-            const listItems: ListItem[] | undefined = await new ListItemRepository().findListItems(checklist.id.toString());
+            const listItems: ListItem[] = await new ListItemRepository().findListItems(checklist.id.toString());
 
-            listItems?.forEach(async (element: ListItem) => {
+            listItems.forEach(async (element: ListItem) => {

```

Comment:
forEach não pode ser assíncrono, pode usar um for of nesse caso.

quiser dar uma olhada:
https://oieduardorabelo.medium.com/javascript-armadilhas-do-asyn-await-em-loops-1cdad44db7f0